galician/dixital/dixital.md

```diff
@@ -0,0 +1 @@
+Polas rañuras da porta alumeaman alternativamente raiolas de luz artificial, as veces vermellas, outras azuis. A viaxe espacial a través do canal dixital comezara, a información recollida dos cúmulos estelares iniciaba unha nova era.
```

Comment:
ok

On Sat, Jan 9, 2016 at 10:50 AM, Casey Collaborator <
notifications@github.com> wrote:

> In galician/dixital/dixital.md
> <https://github.com/udacity/create-your-own-adventure/pull/3207#discussion_r49262540>
> :
>
> > @@ -0,0 +1 @@
> > +Polas rañuras da porta alumeaman alternativamente raiolas de luz artificial, as veces vermellas, outras azuis. A viaxe espacial a través do canal dixital comezara, a información recollida dos cúmulos estelares iniciaba unha nova era.
>
> This line is longer than 120 characters. Could you break it up into
> multiple lines? That way it will be easier to read the raw files or view
> diffs on the command line. No need to add blank lines in between, just
> press enter somewhere in the middle of the line.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/udacity/create-your-own-adventure/pull/3207/files#r49262540>
> .
>